skills/skill-eventmanager/__init__.py

```diff
@@ -20,15 +20,16 @@ async def eventmanager(self, event: Request):
         for alert in payload["alerts"]:
             if alert["status"].upper() == "RESOLVED":
                 continue
-            msg = ""
+            template = "New alert: {} in {}"
```

Comment:
In eventmanager we don't have alerts. Anyway, i think we should keep the same format as in alertmanager, keep msg definition as is and add origin to the template.